From fc38f6d2ad9e85a8997ae413ef8120241a40c07b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mos=C3=A8=20Giordano?= <mose@gnu.org>
Date: Mon, 2 May 2022 22:33:52 +0100
Subject: [PATCH 1/1] User lowercase name for Windows library

Most Windows on users would have a case-insensitive filesystem, but the case is
relevant when compiling on a case-sensitive filesystem, which is usually the
case when cross-compiling with MinGW on Linux, and MinGW consistently uses
lowercase names for libraries and header files.
---
 CMakeModules/FindLIBXML.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeModules/FindLIBXML.cmake b/CMakeModules/FindLIBXML.cmake
index 37aef46e4..8e6318138 100644
--- a/CMakeModules/FindLIBXML.cmake
+++ b/CMakeModules/FindLIBXML.cmake
@@ -68,7 +68,7 @@ if (EXISTS ${ZLIB_LIBRARY})
 set(ADDITIONAL_LIBS "${ADDITIONAL_LIBS}ZLIB::ZLIB;")
 endif()
 if (WIN32)
-set(ADDITIONAL_LIBS "WS2_32.lib;${ADDITIONAL_LIBS}")
+set(ADDITIONAL_LIBS "ws2_32.lib;${ADDITIONAL_LIBS}")
 endif()
 
 if(NOT TARGET LIBXML::LIBXML)
-- 
2.36.0

